pages/linux/gsettings.md

```diff
@@ -0,0 +1,32 @@
+# gsettings
+
+> Query and modify dconf settings with schema validation.
```

Comment:
I'm not sure if we even need to be that specific. Reading that it's not obviously clear what it does (in my opinion). How about just:
> GSettings configuration tool